python/cli/prelude_cli/views/scm.py

```diff
@@ -851,31 +851,50 @@ def put_report(controller, report_data, report_file, report_id):
     "--sort_by",
     "-s",
     help="sort method",
-    type=click.Choice(["a-z", "z-a", "0-9", "9-0"]),
+    type=click.Choice(["group_asc", "group_desc", "count_asc", "count_desc"]),
     default="9-0",
 )
 @click.option(
-    "--scopes",
-    "-c",
-    help="comma-separated list of scope to value pairs, i.e. instances/control=1,instances/platform=windows",
+    "--display_overrides",
+    "-d",
+    help="display overrides in JSON format",
     default=None,
     type=str,
 )
 @click.option(
     "--odata_filter", "-f", help="OData filter string", default=None, type=str
 )
+@click.option(
+    "--scopes",
+    "-c",
+    help="scope-value map in JSON format",
+    default=None,
+    type=str,
+)
 @click.pass_obj
 @pretty_print
 def get_chart_data(
-    controller, scm_category, group_by, group_limit, sort_by, scopes, odata_filter
+    controller,
+    scm_category,
+    group_by,
+    group_limit,
+    sort_by,
+    display_overrides,
+    odata_filter,
+    scopes,
 ):
     """Get chart data for SCM reports"""
+    if display_overrides:
+        display_overrides = json.loads(display_overrides)
+    if scopes:
+        scopes = json.loads(scopes)
```

Comment:
Possible nit: 

I wonder if this reads better than

 `display_overrides = json.loads(display_overrides) if display_overrides else None`

What would mahina say?